src/Symfony/Component/Translation/Translator.php

```diff
@@ -491,4 +492,19 @@ protected function assertValidLocale($locale)
             throw new \InvalidArgumentException(sprintf('Invalid "%s" locale.', $locale));
         }
     }
+
+    /**
+     * @see WarmableInterface
+     *
+     * @param string $cacheDir only used for interface compatibility, $this->cacheDir is already defined
+     */
+    public function warmUp($cacheDir)
+    {
+        if ($this->locale !== null) {
```

Comment:
Sorry, I edited my comment, I meant ```translations``` subfolder and not annotations. Anyway, why should I check ```$this->cacheDir``` ? ```loadCatalogue``` is already doing this check for me. Right ?